spec/models/iiif_presentation_spec_v3.rb

```diff
@@ -170,7 +170,7 @@
       expect(iiif_presentation.manifest["metadata"].class).to eq Array
       expect(iiif_presentation.manifest["metadata"].first.class).to eq Hash
       expect(iiif_presentation.manifest["metadata"].first["label"]["en"].first).to eq "Creator"
-      expect(iiif_presentation.manifest["metadata"].first["value"]['none'].first).to include "Morris & Co. (London, England)"
+      expect(iiif_presentation.manifest["metadata"].first["value"]['none'].first).to include "Morris &amp; Co. (London, England)"
```

Comment:
Not sure why this changed, the data should be text formatted, not HTML.

Reply:
This is a bug and I didn't realize.  I'll take this back to in progress til it's fixed.

![image](https://github.com/yalelibrary/yul-dc-management/assets/36549923/9fe87413-5738-450d-8c91-986a15c1e61e)

Reply:
Will address in separate ticket.